src/algorithms/fardetectors/FarDetectorTrackerClusterConfig.h

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2023, Simon Gardner
+
+#pragma once
+
+namespace eicrecon {
+  struct FarDetectorTrackerClusterConfig {
+
+    std::string readout{""};
+    std::string moduleField{"module"};
+    std::string layerField{"layer"};
+    std::string xField{"x"};
+    std::string yField{"y"};
+
+    int n_module{2};
+    int n_layer{4};
+
+    double time_limit{10};
+
```

Comment:
It would be useful to add documentation in the comments as to what these values are representing, and what units they are in (`time_limit`).